patroni/utils.py

```diff
@@ -33,10 +33,15 @@ def calculate_ttl(expiration):
     return int((expiration - now).total_seconds())
 
 
+def set_ignore_sigterm(value=True):
+    global __ignore_sigterm
+    __ignore_sigterm = value
+
+
 def sigterm_handler(signo, stack_frame):
     global __ignore_sigterm
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using global for '__ignore_sigterm' but no assignment is done](https://www.codacy.com/app/cyberdemn/patroni/file/2272668556/issues/source?bid=3127594&fileBranchId=3185826#l42)